gulp-tasks/reference-docs/get-source-code.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Add the JSDoc description to this file and include at least `@fileoverview` and `@author`

Reply:
Done.